lib/iris/tests/unit/analysis/test_Aggregator.py

```diff
@@ -315,6 +319,37 @@ def test_kwarg_pass_through_combined_kwargs(self):
         expected_kwargs.update(call_kwargs)
         lazy_func.assert_called_once_with(data, axis, **expected_kwargs)
 
+class Test_aggregated_by(tests.IrisTest):
+    def setUp(self):
+        # This sets up a cube with a time coordinate containing two years and
+        # two months.
+        # This will test the ability of the aggregator to correctly calculate
+        # the order of months in the new coordinate array.
+        latitude = DimCoord(np.linspace(-90, 90, 4),
+                            standard_name='latitude',
+                            units='degrees')
+        longitude = DimCoord(np.linspace(45, 360, 8),
+                             standard_name='longitude',
+                             units='degrees')
+        time_unit = unit.Unit('days since epoch',
```

Comment:
I meant to hard code the date, so that when this test is read it is immediately clear what the start month is:

```python
>>> import cf_units
>>> cf_units.Unit('days since epoch', calendar='360_day')
Unit('days since 1970-01-01 00:00:00', calendar='360_day')
```